packages/@aws-cdk/aws-cloudwatch/lib/alarm.ts

```diff
@@ -74,31 +80,6 @@ const OPERATOR_SYMBOLS: {[key: string]: string} = {
   LessThanOrEqualToThreshold: '<=',
 };
 
-/**
- * Specify how missing data points are treated during alarm evaluation
- */
-export enum TreatMissingData {
```

Comment:
I don't think this move makes sense. This is directly related to creating an alarm.